docs/datasets_and_tables/navigating_dbt_docs.md

```diff
@@ -0,0 +1,37 @@
+(navigating-dbt-docs)=
+# `Navigating the dbt Docs`
```

Comment:
Is there a reason to have this be `code style`? Wondering if the `s should be removed

Reply:
@lauriemerrell yeah this was dum. it just made it stand out in the left nav as different from the rest of the sections but is unnecessary/misleading